package.json

```diff
@@ -10,10 +10,12 @@
     "postinstall": "nuxt prepare"
   },
   "dependencies": {
+    "clsx": "^2.1.1",
     "mapbox-gl": "^3.3.0",
     "nuxt": "^3.11.2",
     "nuxt-svgo": "^4.0.1",
     "prettier": "^3.2.5",
+    "tailwind-merge": "^2.3.0",
```

Comment:
It looks like some of these could be `devDependencies` (e.g. `prettier` and so on) - doesn't have any functional impact but it's a good habit to follow the convention for these